apps/meteor/app/lib/server/functions/addUserToRoom.ts

```diff
@@ -85,16 +85,16 @@ export const addUserToRoom = async function (
 				},
 			};
 			if (room.teamMain) {
-				await Message.saveSystemMessage('added-user-to-team', rid, userToBeAdded.username, userToBeAdded, extraData);
+				await Message.saveSystemMessage('added-user-to-team', rid, userToBeAdded.username || '', userToBeAdded, extraData);
```

Comment:
throw an error if username is empty. saving a message with an invalid user is not a valid planned case